js/pie_utils.js

```diff
@@ -69,23 +61,35 @@ function pieData(geodata) {
   var pieAll = [];
   
     for (var i=0; i<geodata.features.length; i++) {
-        pieAll.push( {
-          piechartdata: [
-            {"category": "thermoelectric", 
-                "value": geodata.features[i].properties.thermoelectric},
-            {"category": "publicsupply", 
-                "value": geodata.features[i].properties.publicsupply},
-            {"category": "irrigation", 
-                "value": geodata.features[i].properties.irrigation},
-            {"category": "industrial", 
-                "value": geodata.features[i].properties.industrial},
-            {"category": "other", 
-                "value": geodata.features[i].properties.other}],
-          piechartmeta: [
-            geodata.features[i].geometry.coordinates,
-            geodata.features[i].properties.total
-          ]
-        });
+      properties = geodata.features[i].properties;
+      var proj = projection(geodata.features[i].geometry.coordinates);
+      pieAll.push( {
```

Comment:
I really like the clean up and improvements here! Having `pie` called here (and just once) rather than when we are adding the data. Plus, including the total values in each one, rather than my wonky attribute solution.